brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1272 Fix: Coming Soon Mode - CSS not loading correctly

### Description
Coming Soon Mode: Desktop CSS Carried in Responsive Device

### Screenshots
Issue - https://d.pr/v/HAkWvY
Fix - https://share.getcloudapp.com/Wnu7yLme

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?

- Create a page
- Set the page for coming soon mode
- Add a form using Fluent Form Styler
- In Fluent forms add a container and add fields inside that - https://share.getcloudapp.com/9Zuomwy4 
- Enable coming soon mode
- Check in the incognito window to reproduce this issue

### Checklist:
